index.html

```diff
@@ -3885,6 +3885,29 @@ <h3>JSON-LD</h3>
 base media type, `application/vc+ld+json`.
         </p>
 
+        <p>
+As elaborated upon in Section <a href="#static-processing"></a>, some software
+applications might not perform dynamic JSON-LD processing, which utilizes
+features such as JSON-LD compaction and expansion. Static processors will
+elect to, for example, reject unrecognized URL values in the
+`@context` property. <a>Conforming document</a> authors are also advised that
+usage of certain dynamic JSON-LD processor features might reduce
+interoperability. These features include:
+        </p>
+
+        <ul>
+          <li>
+Use of JSON-LD keywords in the `@context` value that globally modify document
+value processing, such as global settings of `@base` and `@language`
+          </li>
+          <li>
+Using full URLs for JSON-LD terms and types instead of the compact form of those
+values defined in application-specific JSON-LD context files
```

Comment:
```suggestion
Use of full URLs for JSON-LD terms and types, instead of the compact forms of
those values as are defined in application-specific JSON-LD context files
```

Reply:
We can also add another interop problem, specifically

"Use of full URLs for JSON-LD terms and types, when the compact forms of
these values are not defined in application-specific JSON-LD context files"

This is acknowledging that an application may do this but it could cause interop problems.

Reply:
@OR13 are you happy for the above to be added